server/user-groups.ts

```diff
@@ -2,6 +2,7 @@ import {FS} from '../lib/fs';
 
 export type GroupSymbol = '~' | '&' | '#' | '★' | '*' | '@' | '%' | '☆' | '+' | ' ' | '‽' | '!';
```

Comment:
Should ~ still be included here?

Reply:
I feel like a lot of side servers still use it; keeping it in would save them the trouble of having to patch it back in. On the other hand, PS main doesnt use it at all, so keeping it in as a legacy rank does feel a bit odd.

Reply:
I think this is fine as-is.